src/main/java/ui/BaseController.java

```diff
@@ -0,0 +1,91 @@
+package ui;
+
+import data.Database;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+/**
+ * Created by Ari on 2/14/17.
+ */
+abstract class BaseController
+{
+	private static Stage stage;
+	private boolean currentSceneSupportsHC = true;
+	private String[] highContrastBlackList = {Paths.LOGIN_FXML, Paths.DIRECTORY_FXML};
```

Comment:
I'm guessing this could be static, right? If that's the case, the initialization for these should go in `static { } `, I'd think.